ts/src/zebpayspot.ts

```diff
@@ -0,0 +1,1204 @@
+            'open': undefined,
+            'close': close,
+            'last': last,
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': percentage,
+            'average': undefined,
+            'baseVolume': undefined,
+            'quoteVolume': undefined,
+            'markPrice': undefined,
+            'info': ticker,
+        }, market);
+    }
+
+    parseOrder (order: Dict, market: Market = undefined): Order {
+        //
+        //      {
+        //          "orderId": "64507d02921f1c0001ff6892-123-zeb",
+        //          "datetime": "2025-03-14T14:34:34.4567",
+        //          "timestamp": 1741962557553,
+        //          "status": "open",
+        //          "symbol": "BTCINR",
+        //          "type": "market",
+        //          "side": "buy",
+        //          "price": "700000",
+        //          "avgExecutedPrice": "0",
+        //          "filled": null,
+        //          "remaining": 0.002,
+        //          "filledQty": "0",
+        //          "openQty": 0.002,
+        //          "feeCurrency": "INR",
+        //          "fees": "0",
+        //          "tds": "0",
+        //          "tax": "0"
+        //      }
+        //
+        const marketId = this.safeString (order, 'symbol');
+        market = this.safeMarket (marketId, market);
+        const symbol = market['symbol'];
+        const type = this.safeString (order, 'type');
+        const timestamp = this.safeInteger (order, 'createdAt');
+        const datetime = this.iso8601 (timestamp);
+        const price = this.safeString (order, 'price');
+        const side = this.safeString (order, 'side');
+        const amount = this.safeString (order, 'origQty');
+        const filled = this.safeString (order, 'filledQty');
+        const remaining = this.safeString (order, 'openQty');
+        const orderId = this.safeString (order, 'orderId');
+        const timeInForce = undefined;
+        const status = this.safeString (order, 'status');
+        return this.safeOrder ({
+            'id': orderId,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': timeInForce,
+            'postOnly': undefined,
+            'reduceOnly': undefined,
+            'side': side,
+            'amount': amount,
+            'price': price,
+            'triggerPrice': undefined,
+            'cost': undefined,
+            'filled': filled,
+            'remaining': remaining,
+            'timestamp': timestamp,
+            'datetime': datetime,
+            'fee': undefined,
+            'status': status,
+            'info': order,
+            'lastTradeTimestamp': undefined,
+            'lastUpdateTimestamp': undefined,
+            'average': undefined,
+            'trades': undefined,
+        }, market);
+    }
+
+    orderRequest (symbol, type, amount, request, price = undefined, params = {}) {
+        const upperCaseType = type.toUpperCase ();
```

Comment:
don't you support things like `clientOrderId`, timeInForce, postOnly, etc? we might need to unify these deatures here